src/adaptation/icap/ModXact.cc

```diff
@@ -1292,7 +1292,7 @@ Adaptation::Icap::ModXact::~ModXact()
 // internal cleanup
 void Adaptation::Icap::ModXact::swanSong()
 {
-    debugs(93, 5, "swan sings" << status());
+    debugs(93, 5, "swan sings " << status());
```

Comment:
method name "swanSong" makes the content text redundant.
```suggestion
    debugs(93, 5, status());
```